src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: block;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  color: #000;
+  display: flex;
+  white-space: nowrap;
+  align-items: center;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding